cloudbridge/cloud/providers/gce/resources.py

```diff
@@ -1466,7 +1508,12 @@ def zone_id(self):
 
     @property
     def source(self):
-        return self._volume.get('sourceSnapshot')
+        return (self._volume.get('sourceSnapshot') or
```

Comment:
This should return a CloudBridge Snapshot or Image object, instead of the unwrapped version.